config

```diff
@@ -2,17 +2,21 @@
 
 TARGET=${TARGET:-"centos"}
 
-export NAMESPACE=${DOCKER_NAMESPACE:-"rhchestage"}
+export NAMESPACE="rhchestage"
 
-if [ $TARGET == "rhel" ]; then
+if [ "$TARGET" == "rhel" ] || [ "$TARGET" == "rh-integration-test" ]; then
```

Comment:
Actually I think that the original intent of @riuvshin was to use variable `TARGET` to indicate the Dockerfile to use for the docker build (`rhel` or `centos`). Probably the variable should have been called differently. Anyway here you are using this variable for something different. This is difficult to understand for someone that read this code without other explanations and can easily lead to errors. As I commented above you could use a `PRCheckBuild` variable and verify if it's set to `true` here (and below).

Reply:
That will require complex changes to the ci.centos.org yaml.
There is no logic in place that would count with multiple extra variables set in the templates.
Other than that, I'm using `&job_template_defaults` and not `&job_template_build_defaults` which contains the `$TARGET` logic. I don't know how much could that change of templates influence my pr-check.

Reply:
For example, job_template_build_defaults doesn't contain our logic for artifact-ing the runs.
It doesn't save surefire reports.
I could theoretically circumvent that by adding `TARGET="rhel" PR_CHECK_BUILD="true"` into our `ci_cmd`
https://github.com/openshiftio/openshiftio-cico-jobs/blob/master/devtools-ci-index.yaml#L2291

Reply:
I would also leave `TARGET=rhel` to indicate, that this is in fact still rhel build of rh-che. And adding another variable indicating, that this is PR check build is fine with me.